ui/app/AppLayouts/Communities/views/EditCommunityTokenView.qml

```diff
@@ -309,7 +309,7 @@ StatusScrollView {
 
             onTextChanged: {
                 const amount = parseInt(text)
-                if (Number.isNaN(amount) || !!errors)
+                if (Number.isNaN(amount) || (!!errors && Object.values(errors).length))
```

Comment:
`errors` seems to be always valid object, so maybe `!!error` check is not needed at all?